oh-my-zsh.sh

```diff
@@ -98,14 +97,14 @@ done
 # Figure out the SHORT hostname
 if [[ "$OSTYPE" = darwin* ]]; then
   # macOS's $HOST changes with dhcp, etc. Use ComputerName if possible.
-  SHORT_HOST=$(scutil --get ComputerName 2>/dev/null) || SHORT_HOST=${HOST/.*/}
+  SHORT_HOST=$(scutil --get ComputerName 2>/dev/null) || SHORT_HOST="${HOST/.*/}"
 else
-  SHORT_HOST=${HOST/.*/}
+  SHORT_HOST="${HOST/.*/}"
 fi
 
 # Save the location of the current completion dump file.
-if [ -z "$ZSH_COMPDUMP" ]; then
-  ZSH_COMPDUMP="${ZDOTDIR:-${HOME}}/.zcompdump-${SHORT_HOST}-${ZSH_VERSION}"
+if [[ -z "$ZSH_COMPDUMP" ]]; then
+  ZSH_COMPDUMP="${ZDOTDIR:-$HOME}/.zcompdump-$SHORT_HOST-$ZSH_VERSION"
```

Comment:
nitpick: this doesn't fail, but in the future it'd be better to keep the braces (`{}`) when using - because it [has a special meaning when using braces](https://zsh.sourceforge.io/Doc/Release/Expansion.html#Parameter-Expansion). This doesn't fail, again, but it makes the code clearer.

Reply:
I pushed the changes. I understand that you are talking about just the braces from `${SHORT_HOST}` and `${ZSH_VERSION}`, or also from `$HOME`?

Reply:
Yeah you got that right. In `$HOME` it's redundant.

Reply:
great, then it's resolved!
I'll have to look up that docs!